tests/test_other.py

```diff
@@ -5098,60 +5098,34 @@ def test_no_nuthin(self):
     # check NO_FILESYSTEM is automatically set, and effective
     def test(opts, ratio, absolute):
       print('opts, ratio, absolute:', opts, ratio, absolute)
-      def get_size(name):
-        return os.stat(name).st_size
       sizes = {}
       def do(name, source, moar_opts):
         self.clear()
-        Popen([PYTHON, EMCC, path_from_root('tests', source), '-o', name + '.js'] + opts + moar_opts).communicate()
-        sizes[name] = get_size(name + '.js')
+        run_process([PYTHON, EMCC, path_from_root('tests', source), '-o', name + '.js'] + opts + moar_opts)
+        sizes[name] = os.path.getsize(name + '.js')
         if os.path.exists(name + '.wasm'):
-          sizes[name] += get_size(name + '.wasm')
+          sizes[name] += os.path.getsize(name + '.wasm')
         self.assertContained('hello, world!', run_js(name + '.js'))
       do('normal', 'hello_world_fopen.c', [])
       do('no_fs', 'hello_world.c', []) # without fopen, we should auto-detect we do not need full fs support and can do NO_FILESYSTEM
       do('no_fs_manual', 'hello_world.c', ['-s', 'NO_FILESYSTEM=1'])
       do('no_nuthin', 'hello_world.c', ['-s', 'EXPORTED_RUNTIME_METHODS=[]'])
       print('  ', sizes)
-      assert sizes['no_fs'] < sizes['normal']
-      assert abs(sizes['no_nuthin'] - sizes['no_fs']) < 30, 'almost no difference between then, now that we export nothing by default anyhow'
-      assert sizes['no_nuthin'] < ratio*sizes['normal']
-      assert sizes['no_nuthin'] < absolute, str(sizes['no_nuthin']) + ' >= ' + str(absolute)
-      assert sizes['no_fs_manual'] < sizes['no_fs'] + 30 # manual can usually remove a tiny bit more
+      self.assertLess(sizes['no_fs'], sizes['normal'])
+      # almost no difference between then, now that we export nothing by default anyhow
+      self.assertLess(abs(sizes['no_nuthin'] - sizes['no_fs']), 30)
```

Comment:
This is still testing abs size difference

Reply:
I'm not sure I feel as confident changing the semantics of this test.

Reply:
Oh agree, was just pointing it out because the phrasing in the PR description made me think you were only commenting about no_nothin_2's use of abs

Reply:
OK.. I fixed up this test too.